wpeverest/everest-forms
# Introduced script to generate a zip package

### All Submissions:






### Changes proposed in this Pull Request:

It introduces a script to generate a zip package for testing, it's a simple script to generate a zip from any branch with any modification. It also helps us attach a zip file in any Pull Request for anyone to test without having to install our development dependencies and build Everest Forms.

**BLOCKER**: This PR depends on `rsync` command which is not available in normal Windows but available in WSL environment. Install `rsync` command on Windows by simply executing installar within Git bash in administrator mode from here https://github.com/shivapoudel/dotfiles-installer/tree/master/rsync

### How to test the changes in this Pull Request:

1. Checkout this branch
2. Run `npm run build:zip`
3. Check for the build zip file.

### Other information:
